### PR TITLE
Help Center: Parse user's message to markdown

### PR DESCRIPTION
**Pull request by renancarvalho**
Related to https://github.com/Automattic/dotcom-forge/issues/9879
Fixes https://github.com/Automattic/dotcom-forge/issues/9879

## Proposed Changes

* Parse message to markdown if a message is being sent from the user to Zendesk

## Why are these changes being made?

* This is necessary to make messages appear nicely in the chat and keep the content of the message just as before.

## Testing Instructions

* Use live link and add this flag `?flags=help-center-experience`
* Start a chat with yourself by asking Wapuu to talk to a human
* Send links and make sure they appear nicely in the message thread
* Check production and compare.
* Images are not being changed, it should work fine

Before: www.google.com is not a link
image

After:  www.wordpress.com is a link
image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: packages/odie-client/src/components/message/message-content.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import clsx from 'clsx';
 import Markdown from 'react-markdown';
 import { useOdieAssistantContext } from '../../context';
-import { Message } from '../../types';
+import { ZendeskMessage, Message } from '../../types';
```
Comment (escapemanuele) [inline packages/odie-client/src/components/message/message-content.tsx]:
```suggestion
import type { ZendeskMessage, Message } from '../../types';
```

Comment (renancarvalho) [reply packages/odie-client/src/components/message/message-content.tsx]: ESLint removes it automatically.

**Review by escapemanuele — CHANGES_REQUESTED** [review]
image

With the parse we're doing, we now strip the message of all the context.

image

In the case it is not showing the button to talk with a HE.

Probably we should also check that the role of the message is not "bot"

File: packages/odie-client/src/components/message/message-content.tsx (hunk 1 above)
Comment (escapemanuele) [reply packages/odie-client/src/components/message/message-content.tsx]: It does not for me :O

**Comment by renancarvalho** [comment]
@escapemanuele great catch! I have addressed both comments. 🙇

**Review by escapemanuele — APPROVED** [review]
Works great, and thanks for addressing my feedback!

**Pull request merged**